phabfive/maniphest.py

```diff
@@ -98,9 +252,20 @@ def create_from_config(self, config_file, dry_run=False):
             rendered_data.append({
                 "title": Template(ticket_data["title"]).render(variables),
                 "description": Template(ticket_data["description"]).render(variables),
-                "projects": ticket_data["projects"],
             })
 
+            r = len(rendered_data)
+            if ("owner" in ticket_data):
+                rendered_data[r-1]["owner"] = ticket_data["owner"]
+            if ("priority" in ticket_data):
+                rendered_data[r-1]["priority"] = ticket_data["priority"]
+            if ("projects" in ticket_data):
+                rendered_data[r-1]["projects"] = ticket_data["projects"]
+            if ("subscribers" in ticket_data):
+                rendered_data[r-1]["subscribers"] = ticket_data["subscribers"]
+            if ("parents" in ticket_data):
+                rendered_data[r-1]["parents"] = ticket_data["parents"]
```

Comment:
Unless it is needed for any of the logic it checks, parenthesis should be avoided as they are not required or needed here

Reply:
Same here, might be unpushed changes but parenthesis still remains